Forms/MainForm.cs

```diff
@@ -62,6 +67,14 @@ public MainForm()
 
 
             Text = AssemblyVersion;
+
+            if (File.Exists(ROOT + "preset_settings" + PresetExt))
```

Comment:
I can't find anywhere where a file with this name is created. Is this used?

Reply:
Oh I meant to rename them all. I'll go back and rename this.